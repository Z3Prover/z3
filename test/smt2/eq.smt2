
; Copyright (c) 2015 Microsoft Corporation


(declare-const a Int)
(declare-fun f (Int) Int)
(simplify (f f))
(simplify (a + 1))
(assert (a = 1))
