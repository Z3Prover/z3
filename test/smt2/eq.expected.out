(error "line 7 column 14: Sort mismatch at argument #1 for function (declare-fun f (Int) Int) supplied sort is (Array Int Int)")
(error "line 8 column 13: no arguments supplied to arithmetical operator")
(error "line 9 column 11: invalid builtin application =")
